build.assets/docker/base.dockerfile

```diff
@@ -59,6 +59,14 @@ RUN curl https://download.docker.com/linux/static/stable/x86_64/docker-$DOCKER_V
     cp /tmp/docker/* /usr/bin && \
     rm -rf /tmp/docker*
 
+# Replace containerd shipped with docker to avoid deadlocks / hangs due to missing exit result
+# https://github.com/gravitational/gravity/issues/1842
+# https://github.com/containerd/containerd/issues/3572#issuecomment-528293369
+RUN curl -L https://github.com/containerd/containerd/releases/download/v1.2.10/containerd-1.2.10.linux-amd64.tar.gz -o /tmp/containerd.tgz && \
```

Comment:
Nit: Should this version come from a Makefile, or not worth it?

Reply:
I figured it just wasn't worth it. It's only relevant in one location, and only on this particular branch.